2.md

```diff
@@ -0,0 +1,4 @@
+Как пользователь, 
```

Comment:
I think there two stories in this one. The first one about sharing and the second one about giving permissions.

Reply:
Splitted into three stories: 2,3,6